src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Microsoft.CodeAnalysis.Razor.Workspaces.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(DefaultNetCoreTargetFrameworks);netstandard2.0;$(DefaultNetFxTargetFramework)</TargetFrameworks>
+    <TargetFrameworks>$(NetVSAndVSCode);netstandard2.0;$(DefaultNetFxTargetFramework)</TargetFrameworks>
```

Comment:
This is outside of the scope of your PR, but I wonder if Razor tooling still needs to compile anything with `netstandard2.0`? I believe we only ship `net8.0` binaries in service hub.